timescaledb/how-to-guides/multinode-timescaledb/multinode-maintenance.md

```diff
@@ -1,4 +1,7 @@
 # Multi-node maintenance tasks
+
+## Maintaining distributed transactions
+
 A distributed transaction runs across multiple data nodes, and can remain in a
```

Comment:
Can we move this para up to L2, so that we don't have a naked heading?

Reply:
Ok, added a para for the original heading.